renovate.json

```diff
@@ -2,13 +2,23 @@
   "commitMessageAction": "update",
   "commitMessagePrefix": "⬆️(dependencies)",
   "commitMessageTopic": "{{depName}}",
-  "enabledManagers": ["npm"],
+  "enabledManagers": ["npm", "setup-cfg"],
   "extends": ["config:base"],
-  "packageRules": [{
-    "groupName": "npm dependencies",
-    "schedule": ["before 7am on monday"],
-    "matchPackagePatterns": ["*"]
-  }],
+  "labels": ["dependencies"],
+  "packageRules": [
+    {
+      "groupName": "npm dependencies",
+      "matchManagers": ["npm"],
+      "schedule": ["before 7am on monday"],
+      "matchPackagePatterns": ["*"]
+    },
+    {
+      "groupName": "python dependencies",
+      "matchManagers": ["setup-cfg"],
+      "schedule": ["before 7am on monday"],
+      "matchPackagePatterns": ["*"]
+    }
+  ],
   "prConcurrentLimit": 2,
   "prHourlyLimit": 1,
```

Comment:
```suggestion
  "prHourlyLimit": 2,
```

I forgot this point on Richie, but I think we should increase the `prHourlyLimit` to `2` as we have now two PRs to create. I'm still investigating why on Richie, no PR has been created yesterday 😕